src/overview/introduction.md

```diff
@@ -6,32 +6,32 @@ TiKV is fully [ACID](https://en.wikipedia.org/wiki/ACID_(computer_science)) comp
-+ The high availability and data consistency of MySQL can't be guaranteed easily based on the traditional Master-Slave replication. 
++ The high availability and data consistency of MySQL can't be guaranteed easily based on the traditional Master-Slave replication.
 
-Although building a proxy based on MySQL directly might be easy at the beginning, we still decided to chose another way, a more difficult path - to build a distributed, MySQL compatible database from scratch.  
+Although building a proxy based on MySQL directly might be easy at the beginning, we still decided to chose another way, a more difficult path - to build a distributed, MySQL compatible database from scratch.
 
-Fortunately, Google met the same problem and had already published some papers to describe how they built [Spanner](http://static.googleusercontent.com/media/research.google.com/en//archive/spanner-osdi2012.pdf) and [F1](https://storage.googleapis.com/pub-tools-public-publication-data/pdf/41344.pdf) to solve it. Spanner is a globally distributed, externally consistent database and F1 is a distributed SQL database based on Spanner. Inspired by Spanner and F1, we knew we could do the same thing. So we started to build TiDB - a stateless MySQL layer like F1. After we released TiDB, we knew we need a Spanner-like database and then we began to develop TiKV. 
+Fortunately, Google met the same problem and had already published some papers to describe how they built [Spanner](http://static.googleusercontent.com/media/research.google.com/en//archive/spanner-osdi2012.pdf) and [F1](https://storage.googleapis.com/pub-tools-public-publication-data/pdf/41344.pdf) to solve it. Spanner is a globally distributed, externally consistent database and F1 is a distributed SQL database based on Spanner. Inspired by Spanner and F1, we knew we could do the same thing. So we started to build TiDB - a stateless MySQL layer like F1. After we released TiDB, we knew we need a Spanner-like database and then we began to develop TiKV.
 
 ## Architecture
 
-Following is the architecture of TiKV: 
+Following is the architecture of TiKV:
 
 ![Architecture](./architecture.png)
 
-According to the picture, there are three TiKV instances in the cluster and each instance uses one [RocksDB](https://github.com/facebook/rocksdb) to save data. On top of RocksDB, we use [Raft](https://raft.github.io/) consensus algorithm to replicate the data. Mostly, we should use at least three replicas to keep data safe and consistent, and these replicas form a Raft group. 
+According to the picture, there are three TiKV instances in the cluster and each instance uses one [RocksDB](https://github.com/facebook/rocksdb) to save data. On top of RocksDB, we use [Raft](https://raft.github.io/) consensus algorithm to replicate the data. Mostly, we should use at lease three replicas to keep data safety and consistence, and these replicas form a Raft group.
```

Comment:
at least

Reply:
I'm very confused. I do not recall editing this file when I made the PR... Thank for you finding this.